src/main/java/com/syntaxphoenix/spigot/smoothtimber/compatibility/logblock/LogBlockDatabaseAccessor.java

```diff
@@ -14,13 +13,18 @@
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.Set;
 
 /**
  * Used for accessing the logblock database, because the api queries to much not needed information,
  * which slow things down enormously.
  */
 public class LogBlockDatabaseAccessor {
 
+    private static final Set<Material> EMPTY_MATERIALS = Set.of(
+            Material.AIR, Material.CAVE_AIR, Material.VOID_AIR
```

Comment:
Use Strings / Dynamic Material loading from Strings instead because CAVE_AIR and VOID_AIR doesn't exist on older versions